### DISCUSSION TITLE
Audi smartphone interface installed. USB is not present -

### DISCUSSION THREAD

**Discussion by spyder76** [Mr-MIBonk/M.I.B._More-Incredible-Bash#328] [General]
Good evening and everyone.
I'm writing this post because I need help.
I have a 2019 a4 b9 android auto and with usb onlycharge. MHI2Q_ER_AUG22_P5092 MU1329 system
I bought a VW group data USB and the related wiring cable.
I followed the procedure for installing the MIB version 3.6.0 and started the all-in-one patch but it sends me this error:
Patch folder/net/mmx/fs/sda0/patches/MHI2Q_ER_AUG22_P5092_MU1329_PACTH NOT faun.
Audi smartphone interface has been installed. USB is not present on media. connecting a smartphone nothing opens
who can help me . Greetings

**Comment by Boso2298** [comment]
Any updates?
